src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageAction.java

```diff
@@ -151,9 +139,11 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                 return createShowPageResult(
                         Const.ViewURIs.INSTRUCTOR_FEEDBACK_RESULTS_BY_GIVER_RECIPIENT_QUESTION, data);
             case "recipient-question-giver":
+                data.initForViewByRecipientQuestionGiver(instructor, data.selectedSection, data.showStats, data.groupByTeam);
                 return createShowPageResult(
                         Const.ViewURIs.INSTRUCTOR_FEEDBACK_RESULTS_BY_RECIPIENT_QUESTION_GIVER, data);
             case "giver-question-recipient":
+                data.initForViewByGiverQuestionRecipient(instructor, data.selectedSection, data.showStats, data.groupByTeam);
```

Comment:
Yes I plan to only set them in the init method eventually when all the viewBy* are done